README.md

```diff
@@ -298,6 +298,7 @@ Please read [CONTRIBUTING](./CONTRIBUTING.md) if you wish to add software.
 * [Vim](http://www.vim.org) - A highly configurable text editor built to enable efficient editing.
 * [Nano](http://https://www.nano-editor.org/) - A popular text editor, by default comes with most Linux distributions.
 * [Visual Studio Code](https://code.visualstudio.com/) - Fast, hackable, multi-platform code editor from Microsoft.
+* [VSCodium](https://vscodium.com/) - Fork of Microsofts Visual Studio Code with Telemetry disabled.
```

Comment:
Thats wrong. It isn't included at all.

Reply:
I thought so too, but they write on their website quote: "Telemetry is disabled".